docs/topics/KotlinBasic.md

```
@@ -0,0 +1,317 @@
+
+ex) 유용한 twoMillion 표현 방식 `2_000_000`
+
+### Boolean
+
+전위연산자(예시 `+ab`)가 메모리 효율이 더 좋다. 스택. pop.
+
+a + b는 중위
+
+| not | ! |
+| --- | --- |
+| 논리합 | || |
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4; Too many cells, extra data will be missing
Table column count

(MD056, table-column-count)